definitions/features/iptables.rb

```diff
@@ -0,0 +1,58 @@
+class Features::Iptables < ForemanMaintain::Feature
+  metadata do
+    label :iptables
+  end
+
+  def perform_action(action_name)
```

Comment:
This feature looks quite "maintenance_mode oriented". May be more flexible if we have more generic feature with e.g. 
`add_chain`, `remove_chain`, `chain_exist?`, `rule_exist?` and some shortcuts like `add_maintenance_mode_chain`, `remove_maintenance_mode_chain`. What do you think?

Reply:
+1  for a suggested change. It becomes more readable than the current change.